userspace/falco/falco.cpp

```diff
@@ -42,26 +42,11 @@ static void display_fatal_err(const std::string &&msg)
 //
 int falco_init(int argc, char **argv, bool &restart)
```

Comment:
WDYT about renaming this to `falco_loop` or `falco_run`?

Reply:
Agreed! 👍